src/util.cpp

```diff
@@ -610,15 +610,15 @@ bool RenameOver(boost::filesystem::path src, boost::filesystem::path dest)
 }
 
 /**
- * Ignores exceptions thrown by Boost's create_directory if the requested directory exists.
+ * Ignores exceptions thrown by Boost's create_directories if the requested directory exists.
  * Specifically handles case where path p exists, but it wasn't possible for the user to
  * write to the parent directory.
  */
-bool TryCreateDirectory(const boost::filesystem::path& p)
+bool TryCreateDirectories(const boost::filesystem::path& p)
 {
     try
     {
-        return boost::filesystem::create_directory(p);
+        return boost::filesystem::create_directories(p);
```

Comment:
Is this utility function needed at all? I think the definition of `fs::create_directories` is already exactly what we want here:  http://www.boost.org/doc/libs/1_61_0/libs/filesystem/doc/reference.html#create_directories

```
Returns: true if a new directory was created, otherwise false.

Throws: As specified in Error reporting.
```